nautobot/dcim/forms.py

```diff
@@ -588,7 +588,7 @@ class RackBulkEditForm(
         required=False,
         widget=StaticSelect2(),
     )
-    u_height = forms.IntegerField(required=False, label="Height (U)")
+    u_height = forms.IntegerField(required=False, label="Height (U)", min_value=1, max_value=100)
```

Comment:
Why this change?

Reply:
Rack.u_height has a min and max validation, adding it here so that we can catch it early